socorro/scripts/fetch_crash_data.py

```diff
@@ -140,7 +142,14 @@ def main(argv):
     if api_token:
         print('Using api token: %s%s' % (api_token[:4], 'x' * (len(api_token) - 4)))
 
-    for crash_id in args.crashid:
+    if args.crashid:
+        crashids_iterator = iter(args.crashid)
```

Comment:
Is it not already a list?

Reply:
I actually have no idea what it is. It comes from args parsing, so I figured I'd wrap it in an iterator figuring can't hurt. But you're right--it's probably fine not to do this.